neuralmonkey/logging.py

```diff
@@ -29,6 +29,8 @@ def _get_time() -> str:
     @staticmethod
     def set_log_file(path: str) -> None:
         """Set up the file where the logging will be done."""
+        if Logging.log_file is not None and not Logging.log_file.closed:
+            Logging.log_file.close()
         Logging.log_file = codecs.open(path, "w", "utf-8", buffering=0)
```

Comment:
Když už jsme u toho, tady má bejt normálně

```python3
Logging.log_file = open(path, "w", encoding="utf-8", buffering=0)
```

mohl bys to prosím upravit? (Ještě odebrat import codecs)